src/pages/verify/e-ids/danish-mitid.mdx

```diff
@@ -158,49 +155,46 @@ Please [see the UX requirements](#mitid-user-interface-requirements) to make sur
 ### Apply for production access
 
 #### Companies registered in Denmark
-If your company is registered in Denmark please follow these steps: 
+If your company is registered in Denmark please follow these steps:
 
 1. Go to the [management dashboard](https://dashboard.criipto.com) and set the environment toggle at the top center to "PRODUCTION".
 2. In the "Identity sources" section, expand the "DK MitID" section
-3. A user with a NemID/MitID employee signature, a socalled "medarbejdersignatur" must click the button and sign in.
-3. Submit the details for your company. Note the following:
-  - The name to show in the MitID login box is the name entered in the "Company alias" box
-  - The "Domain prefix" is typically your company or brand name, e.g. `acme-corp`. Once this registration is completed this will be used to set up your MitID domain, in this case `acme-corp.mitid.dk`.
-4. The information is sent to Nets who will setup your MitID Domain.
-  - Expect this process to take 7-15 work days.
-  - Once your domain has been set up, you can complete the onboarding process (read below)
+3. A user with a MitID employee signature, a socalled "medarbejdersignatur" must click the button and sign in.
```

Comment:
with a MitID employee signature, a socalled "medarbejdersignatur"
->
enrolled in MitID Erhverv